mlflow/projects/utils.py

```diff
@@ -80,3 +100,186 @@ def _is_valid_branch_name(work_dir, version):
         except GitCommandError:
             return False
     return False
+
+
+def generate_env_vars_to_attach_to_run(run):
```

Comment:
Nit: Could we remove this? Doesn't seem to currently be used anywhere, and seems like it'd only be used by the local project execution backend (let me know if I missed something), so would be better not to expose it in this `mlflow.projects.utils` module, given that we expect plugin developers to call functions from this module.

Reply:
Done